Makefile

```diff
@@ -21,7 +21,7 @@ ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.1
 SDK_VERSION=v0.8.1
 
 LD_FLAGS ?= "-X $(VERSION_PKG).version=$(OPERATOR_VERSION) -X $(VERSION_PKG).buildDate=$(VERSION_DATE) -X $(VERSION_PKG).defaultJaeger=$(JAEGER_VERSION)"
-PACKAGES := $(shell go list ./cmd/... ./pkg/... |  grep -v elasticsearch/v1)
+PACKAGES := $(shell go list ./cmd/... ./pkg/...  ./test/... |  grep -v elasticsearch/v1)
```

Comment:
This is causing the CI failure, as it results in the unit tests being performed including the e2e tests.